README.md

```diff
@@ -1,58 +1,41 @@
-# JavaScript Driver for BigchainDB
+# [![js-bigchaindb-driver](media/repo-banner@2x.png)](https://www.bigchaindb.com)
 
-> Inspired by [`js-bigchaindb-quickstart`](https://github.com/sohkai/js-bigchaindb-quickstart) of @sohkhai [thanks]
+> Official Node.js driver for [BigchainDB](https://github.com/bigchaindb/bigchaindb) with some naive helpers to get you on your way making transactions with BigchainDB and Node.js.
 
-> Supports BigchainDB Server v0.10 (Warning: use CORS enabled [branch](https://github.com/bigchaindb/bigchaindb/tree/kyber-master-feat-cors) untill [PR #1311](https://github.com/bigchaindb/bigchaindb/pull/1311) is resolved )
-
-Some naive helpers to get you on your way to making some transactions, if you'd like to use
-[BigchainDB](https://github.com/bigchaindb/bigchaindb) with JavaScript.
-
-Aimed to support usage in browsers or node and ES∞+, so
-you'll probably need a babel here and a bundler there (or use [one of the built versions](./dist)),
-of which I expect you'll know quite well ([otherwise, go check out js-reactor](https://github.com/bigchaindb/js-reactor)).
+[![npm](https://img.shields.io/npm/v/js-bigchaindb-driver.svg)](https://www.npmjs.com/package/js-bigchaindb-driver)
+[![js bigchaindb](https://img.shields.io/badge/js-bigchaindb-39BA91.svg)](https://github.com/ascribe/javascript)
+[![Build Status](https://travis-ci.org/bigchaindb/js-bigchaindb-driver.svg?branch=master)](https://travis-ci.org/bigchaindb/js-bigchaindb-driver)
 
 ## Compatibility
 
-| BigchainDB Server | BigchainDB Javascript Driver |
+| BigchainDB Server | BigchainDB Node.js Driver |
 | ----------------- |------------------------------|
 | `~=0.10.1` | `~=0.1.0` |
 
 
 ## Contents
 
-- [Getting Started](#getting-started)
+- [Installation](#installation)
 - [Usage](#usage)
 - [Speed Optimizations](#speed-optimizations)
 - [Warnings](#warnings)
-- [API](API.md)
-
-## Getting started
+- [npm releases](#npmreleases)
+- [Authors](#authors)
+- [License](#license)
+- [API reference](API.md)
 
-### Install from npm
+## Installation
 
 ```bash
 # install from npm
-npm install js-bigchaindb-driver
-# Install from GitHub - ssh
-npm install git+ssh://github.com/bigchaindb/js-bigchaindb-driver.git
-# Install from GitHub - https
-npm install git+https://github.com/bigchaindb/js-bigchaindb-driver.git
-```
-
-### Import / ES6
-
-```javascript
-// ES6 Browser
-import * as driver from 'js-bigchaindb-driver';
-// ES<<6 Browser
-let driver = require('js-bigchaindb-driver');
-// ES<<6 CommonJS / node
-let driver = require('js-bigchaindb-driver/dist/node');
```

Comment:
always only provide one preferred way of doing things for 80% of use cases. Just like the npm installation instructions, providing one preferred way (`npm install PACKAGE`) is enough.